### DISCUSSION TITLE
Toggle between different outputs/inputs?

### DISCUSSION THREAD

**Discussion by Bunny-Head** [File-New-Project/EarTrumpet#1557] [Completed ideas]
Hello! I was using a separate app for this but would be great to have this option integrated on Trumpet: Hotkeys that rotate between the different avaialbe inputs and/or outputs or even the ability to make a shortcut dedicated button for each one (or just be able to choose between which ones of the availables one to rotate between).

**Comment by riverar** [comment]
@Bunny-Head Check out EarTrumpet Action in dev builds. https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller

**Comment by Bunny-Head** [comment]
Amazing! Working like a charm, thank you!
